src/app/components/views/homepage/homepage.component.ts

```diff
@@ -38,7 +39,8 @@ export class HomepageComponent implements OnInit, OnDestroy {
    * @returns {void}
    */
   public onRequestJobButtonClick(): void {
-    console.log('TODO: awaiting implementation as a future feature.');
+    this.jobCount++;
```

Comment:
Could this become a potential bug if there's an api error? Then if you hit the button again it'll ask for 2 more jobs then you had. Perhaps just pass jobs+1 to call method?

My followup: 
I messed around with this a little and I actually don't think this is an issue. If you click request job it will locally increase the number of expected jobs by one, but if the api fails then it won't display the job list. At that point, the request job button is disabled, so you can't click it again. When the server is functioning properly again and you hit the refresh load button, it will request the new number of jobs (last num + 1), and then show that number of jobs.
All this to say that you can't increment the number of jobs twice, as you will always have to see the new job list before it'll let you request a new job.